hid-tflight4.c

```diff
@@ -219,21 +219,31 @@ static u8 fixed_rdesc[] = {
 0xC0,              // End Collection
 };
 
-static __u8 *tflight_report_fixup(struct hid_device *hdev, 
+
+static int throttle_seesaw_extra_axis = 0;
+module_param(throttle_seesaw_extra_axis, int, 0660);
```

Comment:
```suggestion
module_param(throttle_seesaw_extra_axis, int, 0664);
```
Believe world readable would be safe